pallets/foreign-investments/Cargo.toml

```diff
@@ -11,23 +11,23 @@ version = "1.0.0"
 targets = ["x86_64-unknown-linux-gnu"]
 
```

Comment:
According to my search, this pallet has quite a number of dependencies which don't make use of workspace inheritance.